draft-ietf-protected-audience-key-value-services.md

```diff
@@ -345,14 +349,10 @@ a request message the Key Value Service can consume along with an HPKE context.
    `processed request`.
     1. If decoding fails, return failure.
 7. If no `partitions` are present, return failure.
-8. Set `compression group map` to an empty map.
+8. Set `compressionGroupMap` to an empty map.
 9. For each `partition` in `partitions`:
-   1. Set `partition id` to the value of the `id` field in `partition` if
-      present, otherwise `0`.
-   2. Set `compression group id` to the value of the `compressionGroupid` field in `partition` if
-      present, otherwise `0`.
-   3. Set `compression group map[compression group id]` to `compression group map`.
-10. Return `processed request`, `compression group map`, and `rctxt`.
+   1. Set `compressionGroupMap[compression group id]` to `compressionGroupMap`.
```

Comment:
So the values in `compressionGroupMap` are...itself? Shouldn't it be a list of partitions?

Reply:
Thanks for catching that. The values should be a list of partition Ids.